frontend/integration-tests/tests/kubevirt/kubevirt.login.scenario.ts

```diff
@@ -0,0 +1,14 @@
+import { browser } from 'protractor';
+import { appHost } from '../../protractor.conf';
+import { execSync } from 'child_process';
+import { logIn } from './utils/utils';
+
+describe('Authentication', () => {
```

Comment:
I'm unclear why we need a special login scenario for kubevirt. Can you elaborate?

Reply:
@spadgett We had to add special login because the login page was different in web-ui and openshift/console at certain point in the past, so login scenario from console didn't work.
Although this is not an issue now, I don't see why plugin related test suites should be running whole console login scenario.
I would prefer having some prerequisite that handles the login quickly rather than executing the whole console login scenario that is executed as a part of different suite anyway.